tests/Makefile

```diff
@@ -59,6 +59,13 @@ TARGETS += uint128-intrinsics-test.exe
 endif
 endif
 
+# AESNI
+ifneq ($(COMPILE_AESNI),)
+CFLAGS += -DHACL_CAN_COMPILE_AESNI
+else
+TARGETS := $(filter-out aes128-ctr32-ni-test.exe, $(TARGETS))
```

Comment:
nit: just put `%-ni-test.exe` here so that future tests are filtered out too

Reply:
Done!